front/src/pages/nft-catalog/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { useStyles } from "./NFTCatalogStyles";
+import FilterSection from "../../components/filters/FilterSection";
+import NFTCatalogWrapper from "../../components/nft-catalog-wrapper/NFTCatalogWrapper";
+import GCContainer from "../../gc-components/GCContainer";
+import GCSegmentedControl from "../../gc-components/segmented-control/GCSegmentedControl";
+import Sorting from "../../components/sorting/Sorting";
+
+const NFTCatalog = () => {
+    const { classes } = useStyles();
+
+    const [layout, setLayout] = useState("catalogOfThree");
+
+    return (
+        <GCContainer>
```

Comment:
This container serves a different purpose when compared to PageWrapper.
PageWrapper is used only for layout purposes, so for example, we can see header on every page, while we might not need container on every page.

We can possibly merge those two and have a container around {children} within PageWrapper container, but then we lose option not to use container on specific page when we might need. Both approaches are good from my perspective.